common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -20,7 +18,7 @@
               % else:
               ${label}
```

Comment:
@ahsan-ul-haq I don't think we want the``` {label}``` or the ```{value}``` included in the status at all, so i would take this conditional out